File: python/7 kyu/248.py
# https://www.codewars.com/kata/563fb342f47611dae800003c/train/python

# Passed

def trim(phrase, size):
    SUFFIX = "..."    
    if len(phrase) <= size:
        return phrase
    elif size <= 3:
        result = phrase[:size] + SUFFIX
    else:
        result = phrase[:size - 3] + SUFFIX
    
    return result

output = trim("Creating kata is fun", 14)
print(output)

File: nodejs/7 kyu/248.js
// https://www.codewars.com/kata/561046a9f629a8aac000001d/train/javascript

// Passed

function matchArrays(v, r) {
  let numberOfCommonElements = 0;
  for (const item of v) {
    if (r.includes(item)) {
      numberOfCommonElements++;
    }
  }

  return numberOfCommonElements;
}

const output = matchArrays(
  ["incapsulation", "OOP", "array"],
  ["time", "propert", "paralelism", "OOP"],
);
console.log(output);